### PR TITLE
Add code style check and build script

### PR DESCRIPTION
**Pull request by patrickrobrecht**
- Adds a code style check for CSS files using stylelint. Existing violations are resolved already.
- Adds a code style check for PHP files. Existing violations are fixed or allowed via `phpcs:ignore`.
- Defines a build script for creating the `.min.css` files. The currently versioned `.min.css` files are removed.
- Changes the GitHub Actions to include a `composer install`. In addition the versions of the steps are updated and `readme.txt` is used instead of `README.md`

EDIT: I just saw that #191 and #192 are already open for the same issues. So this PR is an alternative to the other open PRs.

### CODE REVIEW

File: .github/workflows/wordpress-plugin-asset-update.yml [hunk 1]
```diff
@@ -8,10 +8,11 @@ jobs:
     name: Push to master
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
+    - uses: php-actions/composer@v1
```
Comment (stklcode) [inline .github/workflows/wordpress-plugin-asset-update.yml]: Any reason to not use latest v5 of the composer action?
(same for the deployment action)

File: inc/class-cachify.php [hunk 2]
```diff
@@ -696,7 +701,7 @@ public static function process_flush_request( $data ) {
 		}
 
 		/* Check nonce */
-		if ( empty( $_GET['_wpnonce'] ) || ! wp_verify_nonce( $_GET['_wpnonce'], '_cachify__flush_nonce' ) ) {
+		if ( empty( $_GET['_wpnonce'] ) || ! wp_verify_nonce( sanitize_text_field( wp_unslash( $_GET['_wpnonce'] ) ), '_cachify__flush_nonce' ) ) {
```
Comment (stklcode) [inline inc/class-cachify.php]: Nonce generation is pluggable behavior, so it _might_ contain characters that do not pass `sanitize_text_field()`.

Comment (patrickrobrecht) [reply inc/class-cachify.php]: I changed that to `sanitize_key`. There is a long discussion on https://github.com/WordPress/WordPress-Coding-Standards/issues/414 and https://github.com/WordPress/WordPress-Coding-Standards/issues/869 whether WPCS should throw an error at all and whether `sanitize_key` should be used or not. Alternative would be to add another ignore rule for that.

File: .github/workflows/wordpress-plugin-asset-update.yml (hunk 1 above)
Comment (patrickrobrecht) [reply .github/workflows/wordpress-plugin-asset-update.yml]: No, I've updated the workflows to use the latest version of the actions.

File: inc/class-cachify.php (hunk 2 above)
Comment (stklcode) [reply inc/class-cachify.php]: I am aware of the discussions and totally fine with `sanitize_key`.

**Review by stklcode — APPROVED** [review]
Added some minor changes
* .distignore ignored in git export
* WPCS stylelint config 17 -> 19

Good to merge imho.

**Pull request merged**